pkg/controllers/cloud/configmap_controller.go

```diff
@@ -59,6 +59,13 @@ func (r *ConfigMapReconciler) Reconcile(_ context.Context, req ctrl.Request) (ct
 		r.Log.Error(err, "exiting nephe-controller")
 		os.Exit(1)
 	}
+
+	if err := r.setCloudSyncInterval(newControllerConfig.CloudSyncInterval); err != nil {
+		// Unable to continue as interval is not valid.
+		r.Log.Error(err, "exiting nephe-controller")
```

Comment:
I don't think we should be exiting here. For an invalid case, we should revert back to default. Just add a log message conveying that interval is invalid, so reverting to default.

Reply:
This is only for create. If we use the value 30 we can log the error but the ConfigMap contains 30. If the user sees the ConfigMap that is kind of incorrect.

Reply:
@reachjainrahul Should we exit the controller for an invalid interval?

Reply:
reset to default.. dont exit

Reply:
Solved.